cpp/simulations/abm_braunschweig.cpp

```diff
@@ -891,27 +891,27 @@ mio::abm::Simulation create_sampled_simulation(const std::string& input_file, co
     ScalarType exposed_prob = 0.005, infected_no_symptoms_prob = 0.001, infected_symptoms_prob = 0.001,
                recovered_prob = 0.0;
 
-    //Set global infection parameters (similar to infection parameters in SECIR model) and initialize the world
-    auto world = mio::abm::World(num_age_groups);
+    //Set global infection parameters (similar to infection parameters in SECIR model) and initialize the model
+    auto model = mio::abm::Model(num_age_groups);
 
-    set_parameters(world.parameters);
+    set_parameters(model.parameters);
 
-    // Create the world object from statistical data.
-    create_world_from_data(world, input_file, t0, max_num_persons);
-    world.use_migration_rules(false);
+    // Create the model object from statistical data.
+    create_model_from_data(model, input_file, t0, max_num_persons);
+    model.use_mobility_rules(false);
 
     // Assign an infection state to each person.
-    assign_infection_state(world, t0, exposed_prob, infected_no_symptoms_prob, infected_symptoms_prob, recovered_prob);
+    assign_infection_state(model, t0, exposed_prob, infected_no_symptoms_prob, infected_symptoms_prob, recovered_prob);
 
     auto t_lockdown = mio::abm::TimePoint(0) + mio::abm::days(20);
 
     // During the lockdown, 25% of people work from home and schools are closed for 90% of students.
     // Social events are very rare.
-    mio::abm::set_home_office(t_lockdown, 0.25, world.parameters);
-    mio::abm::set_school_closure(t_lockdown, 0.9, world.parameters);
-    mio::abm::close_social_events(t_lockdown, 0.9, world.parameters);
+    mio::abm::set_home_office(t_lockdown, 0.25, model.parameters);
+    mio::abm::set_school_closure(t_lockdown, 0.9, model.parameters);
+    mio::abm::close_social_events(t_lockdown, 0.9, model.parameters);
```

Comment:
Does it make sense to have this here if for Braunschweig one sets much more advanced NPIs later?